labels_new.md

```diff
@@ -0,0 +1,221 @@
+# Labels Proposal
+
+## Goals
+
+* synchronise labels between repos, especially priority labels
+* make colours more useful by making them more uniform
+
+## Current issues
+
+* Priotiry labels settings needs to be synchronised among repos.
+
+* TiDB uses a `category/label` scheme, but with some exceptions
+
+- closed, component, priority, status, type
+
+## Proposal
+
+### Naming Aternatives
+
+* Use `category/label` naming schemes
+* More categories (e.g., a bot category)
+* Allow labels without categories
+* Add emoji to highlight some labels or sub-categories
+* Separate labels for issues vs PRs
+
+### The labels
+`*` indicates I think we should remove an issue, there might be a suggested replacement in `<...>`. `?` means I don't know what the label means.
+
+### Priotity lables
+
+* Priority labels synchronising proposal:
+
+```
+priority/critical-urgent        #cc2400        Highest priority. Must be actively worked on as someone's top priority right now. Release blocker.
+priority/important-longterm     #fe5733        Important over the long term, but may not be staffed and/or may need multiple releases to complete.
+priority/important-soon         #fe5733        Must be staffed and worked on either currently, or very soon, ideally in time for the next release.
+priority/backlog                #feca33        Higher priority than priority/awaiting-more-evidence.
+priority/awaiting-more-evidence #ffe599        Lowest priority. Possibly useful, but not yet enough support to actually get it done.
+needs-priority                  #cdcdcd        Indicates a PR lacks a `priority/foo` label and requires one.
+```
+
+* Repos current settings which need to optimize
+
+- TiDB
+
+```
+priority/individual-blocker
+Priority/P0
+Priority/P1
+priority/P2
+priority/release-blocker
+Priority/Unknown
+```
+
+- TiKV
+
+```
+priority/critical
+priority/high
+priority/low
+priority/release-blocker
+```
+
+- PD
+
+```
+priority/P0
+priority/P1
+priority/P2
+priority/release-blocker        
```

Comment:
I think using P0/P1/P2 is best - it is simple and it is what most of our repos are doing already (and TiKV is using the same number of priorities).

IMO, whether a PR blocks a release is separate from the priority. I.e., a PR might be P0 and release-blocker. OTOH, a PR might block a release, but if the release is not planned to happen for another year, then it might be P2 and a release-blocker.